--- a/src/configure	2016-01-28 01:08:35.405285412 -0700
+++ b/src/configure	2016-01-28 01:12:29.025285323 -0700
@@ -2500,9 +2500,8 @@
     LDFLAGS="$LDFLAGS -L${PS2SDK}/ports/lib"
 fi
 if [ "$a8_target" = "rpi" ]; then
-    CC="${RPI_SDK}/bin/arm-linux-gnueabihf-gcc"
-    CFLAGS="$CFLAGS -I${RPI_SDK}/include -I${RPI_SDK}/include/SDL -I${RPI_SDK}/include/interface/vmcs_host/linux -I${RPI_SDK}/include/interface/vcos/pthreads"
-    LDFLAGS="$LDFLAGS -Wl,--unresolved-symbols=ignore-in-shared-libs -L${RPI_SDK}/lib"
+    CFLAGS="$CFLAGS -I/opt/vc/include -I/opt/vc/include/SDL -I/opt/vc/include/interface/vmcs_host/linux -I/opt/vc/include/interface/vcos/pthreads"
+    LDFLAGS="$LDFLAGS -Wl,--unresolved-symbols=ignore-in-shared-libs -L/opt/vc/lib"
 fi
 if [ "$a8_target" = "android" ]; then
 	CC="arm-eabi-gcc"
--- a/src/platform.h	2016-01-28 01:21:47.805285110 -0700
+++ b/src/platform.h	2016-01-28 01:22:28.905285094 -0700
@@ -94,7 +94,6 @@
 
 #endif /* SUPPORTS_CHANGE_VIDEOMODE */
 
-#ifdef PLATFORM_MAP_PALETTE
 typedef struct PLATFORM_pixel_format_t {
 	int bpp; /* Current bits per pixel */
 	ULONG rmask;
@@ -108,7 +107,6 @@
 /* Convert a table of RGB values, PALETTE, of size SIZE, to a display's native
    format and store it in the lookup table DEST. */
 void PLATFORM_MapRGB(void *dest, int const *palette, int size);
-#endif /* PLATFORM_MAP_PALETTE */
 
 #if defined(SOUND) && defined(SOUND_THIN_API)
 /* PLATFORM_SoundSetup opens the hardware sound output with parameters
